rdr_service/offline/biobank_samples_pipeline.py

```diff
@@ -796,3 +798,52 @@ def in_past_n_days(result, now, n_days, ordered_before=None):
     if max_time:
         return (now - max_time).days <= n_days
     return False
+
+
+# DA-3543: Ordered DNA samples where order was finalized at least a week ago, but haven't been acknowledged by biobank
+# Intended for weekly check/alerts. Orders from more than 30 days ago will not be included in the alerts
+MISSING_STORED_SAMPLES_SQL = f"""
```

Comment:
We try not to write much raw SQL can you convert this a SQLAlchemy method and add it to the dao itself?

Reply:
I am following the raw SQL usage pattern in this `biobank_samples_pipeline` file.  Is that okay?

Reply:
Yes I see it now. It's not ideal but I understand your reasoning.